unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/QuestsHUD/QuestStepUIEntry_Count.cs

```diff
@@ -0,0 +1,36 @@
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class QuestStepUIEntry_Count : MonoBehaviour, IQuestStepUIEntry
```

Comment:
why is it called "_Count" ?

Reply:
Oh I see there is the "count" (still sounds weird to me) and "single" types

Reply:
It counts progress towards a goal. Probably in the near future Steps and Tasks will be merged together